testcontainers/minio.py

```diff
@@ -0,0 +1,67 @@
+from minio import Minio
+from requests import ConnectionError, Response, get
+
+from testcontainers.core.container import DockerContainer
+from testcontainers.core.waiting_utils import wait_container_is_ready
+
+
+class MinioContainer(DockerContainer):
```

Comment:
Sorry, one more. Could you add a few lines of documentation, maybe a doctest example as in #256, and a reference in the documentation [here](https://github.com/testcontainers/testcontainers-python/blob/master/docs/index.rst)? Looks great otherwise!

Reply:
Sure, I just added a lot of docstrings but somehow my doctest is failing in my local environment:

`python -m doctest -v testcontainers/minio.py 
Traceback (most recent call last):
  File "/Users/hedderich/.pyenv/versions/3.7.15/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/Users/hedderich/.pyenv/versions/3.7.15/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/Users/hedderich/.pyenv/versions/3.7.15/lib/python3.7/doctest.py", line 2794, in <module>
    sys.exit(_test())
  File "/Users/hedderich/.pyenv/versions/3.7.15/lib/python3.7/doctest.py", line 2782, in _test
    m = __import__(filename[:-3])
  File "testcontainers/minio.py", line 1, in <module>
    from minio import Minio
ImportError: cannot import name 'Minio' from 'minio' (testcontainers/minio.py)`

Do I need to add my dependencies for doctest somewhere? I have not worked with doctest before.